### ISSUE TITLE
Unexpected results

### ISSUE THREAD

**Issue by Qup42** [ad-freiburg/sparql-conformance#5]
Hey, I am working on QLever Update. [One of the runs](https://qlever.cs.uni-freiburg.de/sparql-conformance-ui/?cur=14479debafb634446e9646afbf78a608fa1a3cb2&prev=7bd2438dffda2c5b63cdc3e47b2649cf205d9d3e) in [a PR of mine](https://github.com/ad-freiburg/qlever/pull/1592/) yielded unexpected results.

- The operation `update via POST directly` fails (passed before). I am unable to spot a difference in the between the previous and current run in the web UI.
- Another operation `sq07 - Subquery with from` succeeds unexpectedly. Nothing related to that operation has been changed in that PR. Looking at the result details it seems to me that different tests are executed.

Can you help me with these problems?

**Comment by SIRDNARch** [comment]
Hi,

- If you look at the reponse we get a **500 Internal Server Error** (in both runs) but expect a **2xx or 3xx** response. The test suite produced false positives. I recently fixed this, which is why the result changed. 
- Since QLever now supports indexing of named graphs I added it to the test suite. This test uses these named graphs and now correctly works.

Sorry for the inconvenience, I am still working on the test suite especially the update part so you will probably run into weird behaviour again, just report them as you did here.
